scripts/icon_studio/controllers/icon_creator/objects.json

```diff
@@ -6,962 +6,1680 @@
+    "nodeString": "  Spot{ translation 0.0370218 0.573555 -1.028 rotation -1.3505393835251057e-16 1.2211986311439658e-16 -1.0 1.96308 controller \"void\" }"
   },
   "projects/robots/clearpath/moose/protos/Moose.proto": {
-    "background": [1.0, 0.0, 1.0],
-    "fields": "translation 0 0.38 0 rotation 0 1 0 5.76"
+    "background": [
+      1.0,
+      0.0,
+      1.0
+    ],
+    "fields": "  translation 0 0 0.38 rotation -6.9110969686190216e-18 9.581733182280534e-18 -1.0 2.0939853071795866",
+    "nodeString": "  Moose{ translation 0 0 0.38 rotation -6.9110969686190216e-18 9.581733182280534e-18 -1.0 2.0939853071795866 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3.proto": {
-    "fields": "rotation -0.2785 0.67622 0.68202 2.58789"
+    "fields": " rotation 0.006654476782697476 -0.005165617502530299 0.999964516537745 1.3049553071795863",
+    "nodeString": " Robotino3{ rotation 0.006654476782697476 -0.005165617502530299 0.999964516537745 1.3049553071795863 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3HokuyoMountingSupport.proto": {
-    "fields": "rotation -0.7746 0.44721 0.44721 1.82348"
+    "fields": "  rotation -9.988609999001447e-06 1.000829999899948e-05 -0.9999999999000309 0.5236053071795866",
+    "nodeString": "  Robotino3HokuyoMountingSupport{ rotation -9.988609999001447e-06 1.000829999899948e-05 -0.9999999999000309 0.5236053071795866 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3Platform.proto": {
-    "fields": "translation 0 -0.16 0 rotation -0.98311 0.12943 0.12943 1.58784"
+    "fields": "  translation 0 0 -0.16 rotation 7.192349998453602e-09 2.073749999554098e-05 0.999999999784978 0.2618053071795865",
+    "nodeString": "  Robotino3Platform{ translation 0 0 -0.16 rotation 7.192349998453602e-09 2.073749999554098e-05 0.999999999784978 0.2618053071795865 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3SickMountingSupport.proto": {
-    "fields": "rotation -0.38269 0 0.92388 3.14159"
+    "fields": "  rotation -0.8628592318041438 -0.35740509601564097 0.3574010960145665 1.7177653071795864",
+    "nodeString": "  Robotino3SickMountingSupport{ rotation -0.8628592318041438 -0.35740509601564097 0.3574010960145665 1.7177653071795864 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3Webcam.proto": {
-    "fields": "translation 0 -0.208 0 rotation -0.18616 0.69475 0.69475 2.7735"
+    "fields": "  translation 0 0 -0.208 rotation -5.94733999978894e-06 5.966989999788244e-06 0.999999999964512 1.0471953071795863",
+    "nodeString": "  Robotino3Webcam{ translation 0 0 -0.208 rotation -5.94733999978894e-06 5.966989999788244e-06 0.999999999964512 1.0471953071795863 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3WebcamMountingSupport.proto": {
-    "fields": "translation 0 -0.211 0 rotation -0.28109 0.6786 0.6786 2.59356"
+    "fields": "  translation 0 0 -0.211 rotation 0.6785976608971787 -0.2810890310957679 -0.6785972083810249 -2.59356",
```

Comment:
this object should have a proper (monotone color) background like everything else